Fixes #5: Added a form with input and the requested functionality

Sakshi-75 opened the pull request:
Fixes #5
Added a form with input field and the requested functionality

YuriDevAT commented on src/index.html:
```diff
@@ -1,2 +1,18 @@
+	<HEAD>
+		<script type = "text/javascript" src="index.js"></script>
+	</HEAD>
+	<BODY>
```
As far as I know, HTML tags are in lowercase, like `<html>` `<body>` and `<head>`.

YuriDevAT commented on src/index.html:
```diff
@@ -1,2 +1,18 @@
 <h1>Semantic HTML tags</h1>
 <h2>How many do you know?</h2>
+<form>
+	<input type="text" id="input1">
```
An input tag always needs a label for accessibility reasons.

YuriDevAT commented on src/index.html:
```diff
@@ -1,2 +1,18 @@
 <h2>How many do you know?</h2>
+<form>
+	<input type="text" id="input1">
+	<button type="submit" onclick = "submitVal()" >Submit</button>
```
Stay consistent by removing unnecessary spaces like in this click function, as well as in the script (please check your code in general for this one).

YuriDevAT commented on src/index.js:
```diff
@@ -0,0 +1,9 @@
+
+function submitVal() {
+	event.preventDefault();
```
Where is event imported? Shouldn't it be like `function submitVal(event)`? Otherwise VSCode shows me this is deprecated.

YuriDevAT commented on src/index.js:
```diff
@@ -0,0 +1,9 @@
+
+function submitVal() {
+	event.preventDefault();
+	var val = document.getElementById("input1").value;
```
Using let instead of var would be preferred.

YuriDevAT commented on src/index.js:
```diff
@@ -0,0 +1,9 @@
+function submitVal() {
+	event.preventDefault();
+	var val = document.getElementById("input1").value;
+	sessionStorage.setItem("val", val);
```
Nice, that you used `sessionStorage` here, I am sure you would have been good to go to implement localStorage to keep the data when refreshing 👍🏽

YuriDevAT reviewed (changes requested):
Hey Sakshi, thank you so much for your contribution. 👏🏽 
I left you a few comments.

Sakshi-75 commented:
Hi @YuriDevAT ,
I've addressed all your comments.
Regarding the event, I tried passing the event as an argument and using that to preventDefault but it was not working. So, I found a work around for it and implemented that.

YuriDevAT commented:
Hey @Sakshi-75. I found a [good article on how to set and get items from localStorage](https://www.tutorialstonight.com/to-do-list-javascript) :) I hope this will help you figuring it out.

Sakshi-75 commented:
Hi @YuriDevAT , are you looking for any particular change in this PR?

YuriDevAT commented:
I thought you are still working on it, since the content does still not remain when refreshing the page, and I did not understand what the `return false` in the form tag does - I thought you were trying something out. That's why I provided a link which could help you figure out how to use localStorage.

Sorry if I misunderstood. Since implementing localStorage is not a requirement anyways I can merge your PR if you confirm that you are finished. 👍🏽

Sakshi-75 commented:
No worries @YuriDevAT 
Yes, for now I'm done and you can go ahead and merge the PR if it looks okay to you.
Maybe I can look into local storage implementation in the future, but for now I won't be able to continue further on that part.

YuriDevAT commented:
Thank you so much for your contribution 🚀, you did great, @Sakshi-75.
